app/javascript/components/zone-form/zone-form.schema.js

```diff
@@ -0,0 +1,113 @@
+import { componentTypes } from '@@ddf';
+
+const createSchema = (props) => ({
+    fields: [
+        {
+            component: componentTypes.SUB_FORM,
+            name: 'zone-information-subform',
+            title: __('Zone Information'),
+            fields: [
+                {
+                    component: componentTypes.TEXT_FIELD,
```

Comment:
This field should be required, if I try to submit the form without filling it, I'm getting an error. You should add these two:
```
  isRequired: true,
  validate: [{ type: validatorTypes.REQUIRED  }]
```

Also you need to import the `validatorTypes` from `@ddf` on line 1.